MonoGame.Framework/Vector2.cs

```diff
@@ -25,7 +25,10 @@ public struct Vector2 : IEquatable<Vector2>
         private static readonly Vector2 unitVector = new Vector2(1f, 1f);
         private static readonly Vector2 unitXVector = new Vector2(1f, 0f);
         private static readonly Vector2 unitYVector = new Vector2(0f, 1f);
-
+        private static readonly Vector2 rightVector = new Vector2(1f, 0f);
+        private static readonly Vector2 leftVector = new Vector2(-1f, 0f);
+        private static readonly Vector2 upVector = new Vector2(-1f,0f);
+        private static readonly Vector2 downVector = new Vector2(1f, 0f);
```

Comment:
right down and up left are the same vectors?

Reply:
Any update @Raader on resolving the identified issue?